customer-commons/src/main/java/no/unit/nva/customer/model/Sector.java

```diff
@@ -0,0 +1,46 @@
+package no.unit.nva.customer.model;
+
+import static java.lang.String.format;
+import static java.util.Arrays.stream;
+import static java.util.stream.Collectors.joining;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import nva.commons.core.SingletonCollector;
+
+public enum Sector {
+    @JsonProperty("Universitet og høyskoler")
```

Comment:
Wouldn't it make more sense to just serialise the sectors as UHI, HEALTH and INSTITUTE? På engelsk, og uten ledetekstene? Gir de noen verdi noensteds? Jeg tror frontend bruker nøkler for slå opp lokaliserte tekster for UI uansett.

Reply:
Absolutely, I am just trying to follow the pattern for publicationWorkflow